recipes/causalml/meta.yaml

```
@@ -0,0 +1,112 @@
+{% set name = "causalml" %}
+{% set version = "0.12.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/causalml-{{ version }}.tar.gz
+  sha256: 1a5216ee58522cce725907b005bf93232eb311929b36a20fbf8ee341b5781ebb
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+    - cython ==0.29.23
```

```suggestion
    - cython
```
Can you loosen this?